tgbotapi.utils/src/commonMain/kotlin/dev/inmo/tgbotapi/extensions/utils/ClassCasts.kt

```diff
@@ -3078,6 +3078,9 @@ inline fun <T> ChatEvent.whenSupergroupChatCreated(block: (SupergroupChatCreated
 @PreviewFeature
 inline fun ChatEvent.asSupergroupChatCreated(): SupergroupChatCreated? = this as? SupergroupChatCreated
 
+@PreviewFeature
+inline fun ChatEvent.asMigratedToSupergroup(): MigratedToSupergroup? = this as? MigratedToSupergroup
```

Comment:
It is required to add also `require*` and `when*` extension :) besides, it must be **after** another `require*` statement

Reply:
Done